index.ts

```diff
@@ -70,37 +68,39 @@ async function advanceRound(message: Discord.Message, user: Discord.User, league
 
     // Unpins *all* other messages by this bot, but until any other messages are expected to be
     // pinned, this is easier than persisting message ids.
-    function unpinOtherMessages(latestMessage: Discord.Message): Promise<Discord.Message[]> {
-        return latestMessage.channel.messages.fetchPinned()
-            .then(pinned_messages => Promise.all(
-                    pinned_messages
-                        .filter((message: Discord.Message) => message.author.id === client.user?.id ?? "")
-                        .filter(message => message.id !== latestMessage.id)
-                        .mapValues((message: Discord.Message) => message.unpin())
-                        .array()
-            ));
+    async function unpinOtherMessages(latestMessage: Discord.Message): Promise<Discord.Message[]> {
+        const pinnedMessages = await latestMessage.channel.messages.fetchPinned();
+        const otherMessages = pinnedMessages
+                .filter((message: Discord.Message) => message.author.id === client.user?.id ?? "")
```

Comment:
Ooh what is `.user?.id`? Is that null-ish handling?

Reply:
Lol I'm pretty sure you wrote this line and I just had to fix a lint error.

Yeah this is null-ish handling in typescript. I don't understand what javascript gets emitted in these cases so I try to avoid it, but I could just try and look it up.